### DISCUSSION TITLE
Where to change API key?

### DISCUSSION THREAD

**Discussion by ahmedelami** [zed-industries/zed#9665] [Feature Requests]
On set-up, I input the Claude AI API key but im assuming Zed only supports gpt keys (would be amazing if it could support other AI's as well) now that that's broken, I cant seem to figure out how to change the API key or where to go to do that, anyone know?

**Comment by Moshyfawn** [comment]
You can find out how to reset the key at https://github.com/zed-industries/zed/discussions/8200#discussioncomment-8558087.

The plan is to support multiple AI providers. For Clauld specifically, you can follow and +👍 #8914.
